src/main.cpp

```diff
@@ -414,9 +407,24 @@ int main(int argc, char* argv[])
     mudlet::debugMode = false;
     FontManager fm;
     fm.addFonts();
-    QString home = QDir::homePath() + "/.config/mudlet";
-    QString homeLink = QDir::homePath() + "/mudlet-data";
-    QFile::link(home, homeLink);
+#ifdef Q_OS_WIN32
+    /*
+     * From Qt Documentation for:
+     * bool QFile::link(const QString &linkName)
+     *
+     * "Note: To create a valid link on Windows, linkName must have a .lnk file
+     * extension."
+     *
+     * Whilst the static form:
+     * [static] bool QFile::link(const QString &fileName, const QString &linkName)
+     * does not mention this particular restriction it is not unreasonable to
+     * assume the same condition applies...
+     */
+    QString homeLink = QStringLiteral("%1/mudlet-data.lnk").arg(QDir::homePath());
```

Comment:
This code then should account for the old `mudlet-data` that we were creating on Windows before and delete it, else Mudlet will pollute the users folder with 2 items now!

Reply:
But was the previous "mudlet-data" link ever _created_? According to the Qt Documentation it shouldn't have worked... :stuck_out_tongue_winking_eye:

Reply:
🤔 Humm,  I recently installed the Windows 3.4.0 binary version on that laptop I took on holiday with me recently - I'll fire 'Doze up on that in the next day and see whether there is a `mudlet.data` link in place on that - mayhaps the Qt Documentation is wrong/out-of-date...

Reply:
Ah, checked it and the "file" was being created, but because it does not have the _extension_ that 'Doze needs to identify the _type_ {*nix-heads will be 🤣 at such a simplistic and easily breakable system as _that_)} it doesn't work as a link: 
![windows home directory with broken mudlet-data link](https://user-images.githubusercontent.com/6163092/30697434-7dc8d1a0-9ece-11e7-8182-bf5435cc1229.png)

![properties of broken mudlet-data link](https://user-images.githubusercontent.com/6163092/30697433-7dc84046-9ece-11e7-95f0-c57abd0bcb88.png)

However merely adding the `.lnk` extension is enough to fix it (as I did on the OS/System I checked on):
![windows home directory with fixed mudlet-data link](https://user-images.githubusercontent.com/6163092/30697767-998613ca-9ecf-11e7-857e-a0bbee010a17.png)

![properties of fixed mudlet-data link](https://user-images.githubusercontent.com/6163092/30697766-99858cfc-9ecf-11e7-8b8c-76379fb8e5fe.png)

I guess that all the previous versions of Mudlet having this flaw means that it is safe to assume that an existing `mudlet-data` in the user's home directory should be renamed to have the required extension but without a working native 'Doze build environment yet I cannot be certain that doing a rename in code {algorithm: IF there a `mudlet-data` file, THEN rename it to `mudlet-data.lnk` ELSE create a link `mudlet-data.lnk` ENDIF} will not have any _side-effects_ in that a `mudlet-data.lnk` to the NON-HIDDEN `\users\`%username%`\.config\mudlet` folder might behave as if it WAS that directory...!

___I would not want to touch a `mudlet-data` file in code in case it makes things worse than just having a rogue directory entry!___ {we should mention it in release notes in the next few 3.x.y releases but if the __Wimdows__ user ever reuses an older one that will just recreate the entry 😊!}